source/system-administrators/deployer/processors-guide.rst

```diff
@@ -232,6 +232,39 @@ Processor that updates the processed commits value with the current commit
 
     - processorName: gitUpdateCommitIdProcessor
 
+.. _deployer-script-processor:
+
+^^^^^^^^^^^^^^^^^^^^^^^
+Groovy Script Processor
+^^^^^^^^^^^^^^^^^^^^^^^
+
+A site-specific custom Groovy processor that can process published content.
+
+**Properties**
+
++------------+-----------+-------------------------------+------------------------------------------------------------+
+|Name        |Required   |Default Value                  |Description                                                 |
++============+===========+===============================+============================================================+
+|scriptPath  ||checkmark||                               |The relative path of the script to execute                  |
++------------+-----------+-------------------------------+------------------------------------------------------------+
+
+  .. note::  The default path scripts are loaded from is ``$CRAFTER_HOME/bin/crafter-deployer/processors/scripts``
+
+
+**Example**
+
+.. code-block:: yaml
+    :linenos:
+    :caption: *Groovy Script Processor*
+
+    - processorName: scriptProcessor
+      scriptPath: 'myscripts/mychanges.groovy'
+
+|
+
```

Comment:
For better guidance to developers I think we should include a small example of a valid Groovy script and a short description of the variables available to use